### ISSUE TITLE
[Feedback] really did the great job about improving LC_API, im not the code worker but nice! if u can, hope u can improve LC_API again

### ISSUE THREAD

**Issue by CoolLKKPS** [DrFeederino/AndysModsPlugin#3]
vanilla compatibility is no longer necessary i think, and i really hate LC_API change the version number; i uninstall LC_API bc those reason, and i already send issue ticket in there, but at that time bc the maintainers no longer maintain that mod and don't have any promise, so i have no choice
*LC_API can modified some behavior about creature spawning i think but i can't get any answer there so, another reason to uninstall LC_API*

**Comment by DrFeederino** [comment]
Hey!

Vanilla is removed from dependencies, I forgot to mention it in changes but it needs to either be disabled/removed as R2Modman doesn't handle such cases where one dependency is dropped => delete it if no other mod depends on it.

Regarding spawning, I can't really tell it messes with spawn rates of enemies. Though, I did glance it at the code quickly. My issue was that with v50 it wouldn't display tips anymore and I need that functionality to tell user about on/off switch state of my mods. I was pleasantly surprised that just re-building it with newer version of the game files did the charm. 

I opened up the [fork ](https://github.com/DrFeederino/LC-API-V50)and contributed my changes, however, I would most probably not be able to maintain it let alone develop it further.
